app/models/post.rb

```diff
@@ -32,12 +34,16 @@ def tags_string
   def tags_string=(value)
     @tags_list = []
```

Comment:
Do you mean that `tags_string=` could just be an `attr_writer` and the processing code could be in the `create_tags_from_tag_string` method?